contracts/ClaimsManager.sol

```diff
@@ -474,6 +475,10 @@ contract ClaimsManager is
             claimState.proposedSettlementAmountInUsd
         );
         clippedPayoutAmountInApi3 = convertUsdToApi3(clippedPayoutAmountInUsd);
+        require(
+            clippedPayoutAmountInApi3 >= minimumPayoutAmountInApi3,
+            "Payout less than minimum"
+        );
```

Comment:
Would it be an issue if this reverts while calling this function statically?

Reply:
You should probably set `minimumPayoutAmountInApi3` to 0 while paying statically

Reply:
Ok that's reasonable. If not I was probably going to suggest having a separate function to return the amount of API3 tokens